experimental/traffic-portal/src/styles.scss

```diff
@@ -263,3 +263,11 @@ form mat-hint.mat-mdc-form-field-hint.input-warning {
 	font-weight: bold;
 	font-size: x-small;
 }
+
+.w-100 {
+	width: 100%;
+}
+
+.h-100 {
+	height: 100%;
+}
```

Comment:
this has a very "embedding CSS inline into HTML" feel to it. In any case where you'd want these, I think a more specific class should probably be applied.

Reply:
removed.